.travis.yml

```diff
@@ -28,5 +90,13 @@ script:
   - make
 
 after_success:
-  - make check
-  - if type cppcheck &> /dev/null ; then cppcheck --error-exitcode=1 --quiet *.h *.c tests/ ; fi
+  - if [ -n "$CHECK" ]; then
+      make distcheck;

```

Comment:
On Travis, and on my own system, I see 3/21 checks that fail when make distcheck is run.

Reply:
> On Travis, and on my own system, I see 3/21 checks that fail when make distcheck is run.

I fixed 2 of the tests in https://github.com/json-c/json-c/pull/500 , the 3rd one is still broken (https://github.com/json-c/json-c/issues/498)

Reply:
3rd one fixed by @hawicz in https://github.com/json-c/json-c/commit/68abf12afab0ff40112e20ef59c8fd865f30c972